app/assets/javascripts/discourse/controllers/preferences/emails.js.es6

```diff
@@ -51,6 +56,25 @@ export default Ember.Controller.extend(PreferencesTabController, {
     { name: I18n.t("user.email_digests.every_two_weeks"), value: 20160 }
   ],
 
+  @computed("model.user_option.email_messages_level")
+  emailMessageLevelAway(level) {
+    return level === 1;
+  },
+
+  @computed("model.user_option.email_level")
+  emailLevelAway(level) {
+    return level === 1;
+  },
+
+  @computed()
+  emailFrequencyInstructions() {
+    if (this.siteSettings.email_time_window_mins)
```

Comment:
Not sure why our es6 linter didn't flag this but we should use braces for if statements.